core/src/main/java/com/klaytn/caver/contract/SendOptions.java

```diff
@@ -102,62 +102,64 @@ public String getValue() {
      * @param from The address of the sender
      */
     public void setFrom(String from) {
-        if(from == null) {
-            throw new IllegalArgumentException("from is missing.");
-        }
+        if(from != null) {
+            if(!Utils.isAddress(from)) {
+                throw new IllegalArgumentException("Invalid address. : " + from);
+            }
 
-        if(!Utils.isAddress(from)) {
-            throw new IllegalArgumentException("Invalid address. : " + from);
+            this.from = from;
         }
-
-        this.from = from;
     }
 
     /**
      * Setter function for gas
      * @param gas The maximum amount of gas the transaction is allowed to use.
      */
     public void setGas(String gas) {
-        //Gas value must be set.
-        if(gas == null || gas.isEmpty() || gas.equals("0x")) {
-            throw new IllegalArgumentException("gas is missing.");
-        }
+        if (gas != null) {
+            if(!Utils.isNumber(gas)) {
+                throw new IllegalArgumentException("Invalid gas. : "  + gas);
+            }
 
-        if(!Utils.isNumber(gas)) {
-            throw new IllegalArgumentException("Invalid gas. : "  + gas);
+            this.gas = gas;
         }
-        this.gas = gas;
     }
 
     /**
      * Setter function for gas
      * @param gas The maximum amount of gas the transaction is allowed to use.
      */
     public void setGas(BigInteger gas) {
-        setGas(Numeric.toHexStringWithPrefix(gas));
+        if(gas != null) {
+            setGas(Numeric.toHexStringWithPrefix(gas));
+        }
     }
 
     /**
      * Setter function for value.
      * @param value The amount of KLAY in peb to be transferred.
      */
     public void setValue(String value) {
-        if(value == null) {
-            throw new IllegalArgumentException("value is missing.");
-        }
+        if(value != null) {
+            if(!Utils.isNumber(value)) {
+                throw new IllegalArgumentException("Invalid value : " + value);
+            }
 
-        if(!Utils.isNumber(value)) {
-            throw new IllegalArgumentException("Invalid value : " + value);
+            this.value = Numeric.prependHexPrefix(value);
         }
 
-        this.value = Numeric.prependHexPrefix(value);
+        this.value = "0x0";
```

Comment:
I think this statement will overwrite `this.value = Numeric.prependHexPrefix(value);`.